bitnami/matomo/values.yaml

```diff
@@ -197,17 +197,23 @@ tolerations: []
 existingSecret: ""
 ## SMTP mail delivery configuration
 ## ref: https://github.com/bitnami/containers/tree/main/bitnami/matomo/#smtp-configuration
+## @param smtpAuth SMPT authentication mechanism (options: Plain, Login, Crammd5)
```

Comment:
```suggestion
## @param smtpAuth SMTP authentication mechanism (options: Plain, Login, Crammd5)
```

Reply:
@fmulero Thanks for the review, I fixed the typo.